lib/persisted-model.js

```diff
@@ -550,6 +577,20 @@ module.exports = function(registry) {
     var typeName = PersistedModel.modelName;
     var options = PersistedModel.settings;
 
+    var configurableVerbs = {
+      updateAttributes: 'patch',
+      updateOrCreate: 'patch',
+      replaceOrCreate: 'put',
+      replaceById: 'put',
+    };
+
+    if (options.updateOnPUT) {
+      configurableVerbs.replaceById = 'post';
+      configurableVerbs.replaceOrCreate = 'post';
```

Comment:
When `updateOnPUT` is set, then you need to change the HTTP path too, not only the verb.

```
updateOnPUT=false

  replaceById     => PUT /:id
  replaceOrCreate => PUT /

updateOnPUT=true

  replaceById     => POST /:id/replaceById
  replaceOrCreate => POST /replaceOrCreate
```